src/Adapter/MSTestAdapter.PlatformServices/AssemblyResolver.cs

```diff
@@ -335,7 +336,7 @@ protected virtual
                     if (EqtTrace.IsInfoEnabled)
                     {
                         EqtTrace.Info(
-                            "AssemblyResolver: {0}: Failed to create assemblyName. Reason:{1} ",
+                            "MSTest.AssemblyResolver.OnResolve: Failed to create assemblyName '{0}'. Reason: {1} ",
```

Comment:
Prefix all assembly resolver messages with `MSTest` and do a little reword to make it easier when debugging logs to understand if it's MSTest or VSTest assembly resolver being triggered.